release/changelog

```diff
@@ -3,8 +3,7 @@
 # changelog generates two files, Contributors and CHANGELOG with all changes
```

Comment:
We could add this changelog file to rubocop ignore so it stays in sync with the one from foreman core.